src/lib/Api/api.ts

```diff
@@ -136,18 +141,36 @@ export default class Api {
   /**
    * Get transfers by block number
    * @param {object} opts object containing all parameters
-   * @param {string} opts.blockHashNumber number of the block to get transfers from
+   * @param {string} opts.blockNumber number of the block to get transfers from
    * @returns {Promise<object>} Transfers list
    */
   async getTransfersByBlockNumber(opts: GetTransfersByBlockNumberOptions): Promise<TransfersDTO> {
-    if (!opts.blockHashNumber) {
+    if (!opts.blockNumber) {
       log.throwMissingArgumentError(Logger.message.invalid_block_number, {
         location: Logger.location.SDK_GETTRANSFERSBYBLOCKNUMBER,
       });
     }
 
     const apiUrl = `${this.apiPath}/nfts/block/transfers`;
-    const { data } = await this.httpClient.get(apiUrl, { blockHashNumber: opts.blockHashNumber });
+    const { data } = await this.httpClient.get(apiUrl, { blockHashNumber: opts.blockNumber });
```

Comment:
cursor should be passed as parameter as well

Reply:
thnx i missed that